bash/runrtqsam

```diff
@@ -39,54 +31,54 @@ shopt -u nullglob
 export SNAPOUT=DUMMY
 
 export SYSUT2=qsam/TESTV.TFV
-${dir}/bash/asmlg qsam/BLDV.MLC trace $1
+bash/asmlg qsam/BLDV.MLC trace $1
 export SYSUT1=qsam/TESTV.TFV
-${dir}/bash/asmlg qsam/CHKV.MLC trace
+bash/asmlg qsam/CHKV.MLC trace
 
 export SYSUT2=qsam/TESTVB.TFV
-${dir}/bash/asmlg qsam/BLDVB.MLC trace $1
+bash/asmlg qsam/BLDVB.MLC trace $1
 export SYSUT1=qsam/TESTVB.TFV
-${dir}/bash/asmlg qsam/CHKVB.MLC trace
+bash/asmlg qsam/CHKVB.MLC trace
 
 export SYSUT2=qsam/TESTVT.TFV
-${dir}/bash/asmlg qsam/BLDVT.MLC trace $1
+bash/asmlg qsam/BLDVT.MLC trace $1
 export SYSUT1=qsam/TESTVT.TFV
-${dir}/bash/asmlg qsam/CHKVT.MLC trace
+bash/asmlg qsam/CHKVT.MLC trace
 
 export SYSUT2=qsam/TESTVL.TFV
-${dir}/bash/asmlg qsam/BLDVL.MLC trace $1
+bash/asmlg qsam/BLDVL.MLC trace $1
 export SYSUT1=qsam/TESTVL.TFV
-${dir}/bash/asmlg qsam/CHKVL.MLC trace
+bash/asmlg qsam/CHKVL.MLC trace
 
 export SYSUT2=qsam/TESTVBL.TFV
-${dir}/bash/asmlg qsam/BLDVBL.MLC trace $1
+bash/asmlg qsam/BLDVBL.MLC trace $1
 export SYSUT1=qsam/TESTVBL.TFV
-${dir}/bash/asmlg qsam/CHKVBL.MLC trace
+bash/asmlg qsam/CHKVBL.MLC trace
 
 export SYSUT2=qsam/TESTVTL.TFV
-${dir}/bash/asmlg qsam/BLDVTL.MLC trace $1
+bash/asmlg qsam/BLDVTL.MLC trace $1
 export SYSUT1=qsam/TESTVTL.TFV
-${dir}/bash/asmlg qsam/CHKVTL.MLC trace
+bash/asmlg qsam/CHKVTL.MLC trace
 
 export SYSUT2=qsam/TESTF.TFV
-${dir}/bash/asmlg qsam/BLDF.MLC trace $1
+bash/asmlg qsam/BLDF.MLC trace $1
 export SYSUT1=qsam/TESTF.TFV
-${dir}/bash/asmlg qsam/CHKF.MLC trace $1
+bash/asmlg qsam/CHKF.MLC trace $1
 
 export SYSUT2=qsam/TESTFL.TFV
-${dir}/bash/asmlg qsam/BLDFL.MLC trace $1
+bash/asmlg qsam/BLDFL.MLC trace $1
 export SYSUT1=qsam/TESTFL.TFV
-${dir}/bash/asmlg qsam/CHKFL.MLC trace
+bash/asmlg qsam/CHKFL.MLC trace
 
 export SYSUT2=qsam/TESTFT.TFV
-${dir}/bash/asmlg qsam/BLDFT.MLC trace $1
+bash/asmlg qsam/BLDFT.MLC trace $1
 export SYSUT1=qsam/TESTFT.TFV
-${dir}/bash/asmlg qsam/CHKFT.MLC trace
+bash/asmlg qsam/CHKFT.MLC trace
 
 export SYSUT2=qsam/TESTFTL.TFV
-${dir}/bash/asmlg qsam/BLDFTL.MLC trace $1
+bash/asmlg qsam/BLDFTL.MLC trace "TIME(30)" $1
 export SYSUT1=qsam/TESTFTL.TFV
-${dir}/bash/asmlg qsam/CHKFTL.MLC trace
+bash/asmlg qsam/CHKFTL.MLC trace
 
 ls qsam/TEST*.TFV
 echo "Verify runqsamrt"
```

Comment:
You might change "runqsamrt" to "runrtqsam".

Reply:
Fixed in 700b53e